pydoctor/templates/apidocs.css

```diff
@@ -157,8 +158,14 @@ tr.private {
 }
 
 .fieldArg {
-    font-weight: bold;
-    color: #c7254e;
+    color: #222222;
```

Comment:
Do we need to override the text color here? The standard text color set by bootstrap is `#333333`, which doesn't look all that different from `#222222`.